scania-digital-design-system/tegel
# test(dropdown): added unit tests

**Describe pull-request**  
Followed the test steps in ticket/excel to test.

**Solving issue**  
Fixes: [CDEP-2960](https://tegel.atlassian.net/browse/CDEP-2960)

**How to test**  
1. Navigate to packages/core.
2. Check the code for tests in dropdown component.
3.  Make sure docker is running.
4. Then run `npm run test` in core to run the tests. Make sure they run.

**Additional context** 
We are still in exploration for good testing.

[CDEP-2960]: https://tegel.atlassian.net/browse/CDEP-2960
